### PR TITLE
Fix zookeeper

### PR DESCRIPTION
**Pull request by dmp42**
cc @willdurand @msabramo

### CODE REVIEW

File: test/fixtures/servers/zookeeper.js [hunk 1]
```diff
@@ -7,9 +7,38 @@
...
+                // And start
+                derefStart.apply(this, passargs);
+            }.bind(this));
+            return this;
+        };
+
+        this.stop = function () {
+            Generic.apply(this, [service, ['stop']]);
+            this.mute = true;
+            this.start();
```
Comment (msabramo) [inline test/fixtures/servers/zookeeper.js]: Why does it call `this.start()`?

File: test/fixtures/servers/zookeeper.js [hunk 2]
```diff
@@ -7,9 +7,38 @@
...
+        Generic.apply(this, [service, ['start']]);
+        // Zookeeper happily outputs to stderr, so, shut it up
+        this.mute = true;
+
+        var derefStart = this.start;
+
+        this.start = function () {
+            var passargs = arguments;
+            exec('which ' + service, function (error/*, stdout, stderr*/) {
+                // Assume brew osx install in case we didn't find it
```
Comment (msabramo) [inline test/fixtures/servers/zookeeper.js]: Nice, thank you for adding this check for `zkServer` instead of `zkServer.sh` on brew osx install!

**Comment by msabramo** [comment]
Tests are working much better on my Mac now. Thanks!

File: test/fixtures/servers/zookeeper.js (hunk 1 above)
Comment (dmp42) [inline test/fixtures/servers/zookeeper.js]: I know - confusing enough...

There is two different process here - `zkserver start` and `zkserver stop` and both need to be "started"...

**Pull request merged**